.github/workflows/publish-release.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.MAVEN_SIGNING_PASSWORD }}
 
       - name: Upload release (Windows)
-        run: ./gradlew publishMingwPublicationToMavenRepository --no-daemon --no-parallel
+        run: ./gradlew publishMingwX64PublicationToMavenCentralRepository --no-daemon --no-parallel
```

Comment:
Was a mistake I made in the initially added file in the mad repo. That's why there is a `0.1.1` release which has the missing windows artifacts where publishing failed. The snapshot workflow was already correct.